src/scan-iterator.ts

```diff
@@ -60,15 +61,60 @@ export class ScanResult<K> implements AsyncIterable<JSONValue> {
    * [[ReadTransaction.scan|scan]] is over an index the key is a tuple of
    * `[secondaryKey: string, primaryKey]`
    */
-  entries(): AsyncIterableIterator<[K, JSONValue]> {
-    return this._newIterator(ENTRY);
+  entries(): AsyncIterableIteratorToArrayWrapper<[K, JSONValue]> {
+    return new AsyncIterableIteratorToArrayWrapper(this._newIterator(ENTRY));
+  }
+
+  /** Returns all the values as an array. Same as `values().toArray()` */
+  toArray(): Promise<JSONValue[]> {
```

Comment:
@aboodman This returns same as `[Symbol.asyncIterator]()` for consistency. Maybe it would have been better to default to entries? When I first wrote this I felt like scan result was more similar to a List and not a Map